yew/Cargo.toml

```diff
@@ -120,6 +120,7 @@ trybuild = "1.0"
 rustversion = "1.0"
 rmp-serde = "0.14.0"
 bincode = "1"
+boolinator = "2.4.0"
```

Comment:
I don't think there's anything we need `boolinator` for to test, is there? Let's not add a dependency just for that...